cmake/Modules/OpmInit.cmake

```diff
@@ -12,6 +12,50 @@
 # This module should be the first to be included in the project,
 # because most of the others (OpmXxx.cmake) use these variables.
 
+# for CMake >= 3.0, we need to change a few policies:
+#
+#   - CMP0026 to allow access to the LOCATION target property
+#   - CMP0048 to indicate that we want to deal with the *VERSION*
+#     variables ourselves
+#   - CMP0064 to indicate that we want TEST if conditions to be evaluated
+macro(OpmSetPolicies)
+  message("Setting policies")
+  if (POLICY CMP0026)
+    cmake_policy(SET CMP0026 OLD)
+  endif()
+
+  if (POLICY CMP0048)
+    cmake_policy(SET CMP0048 OLD)
+  endif()
+
+  if(POLICY CMP0064)
+    cmake_policy(SET CMP0064 NEW)
+  endif()
+
+  # set the behavior of the policy 0054 to NEW. (i.e. do not implicitly
+  # expand variables in if statements)
+  if (POLICY CMP0054)
+    cmake_policy(SET CMP0054 NEW)
+  endif()
+
+  # include special
```

Comment:
these are rather pointless. we already rely on newer functionality (in particular to set c++17). but can be stripped out separately, of course.

Reply:
Won't there be warings for older versions? We only require 2.8. Not sure whether that is sufficient though.

CMP0026 is still used to set ${opm}_LIBRARY which seems to be used quite extensively in the still complicated CMake code.

I did set CMP0048 to NEW now. The rest has to be done in a different PR.

Reply:
we only require 2.8 but that is bogus these days because of the use of CMAKE_CXX_STANDARD etc. that came in somewhere around 3.0 and you need even newer than that to set it to 17. but anyways, we will address that later, was just a drive-by comment.

Reply:
BTW it is not pointless. The documentation says for CMP0026-0048
```
This policy was introduced in CMake version 3.1|3.0.
CMake version 3.13.4 warns when the policy is not set and uses
``OLD`` behavior.  Use the ``cmake_policy()`` command to set
it to ``OLD`` or ``NEW`` explicitly.
```

My interpretation is that we get a warning if we do not set anything (and I experienced that on Debian)

Reply:
my point was trying to do something for cmake 2.8, not that what we do is pointless as such.